src/components/scheduler.ts

```diff
@@ -0,0 +1,172 @@
+import DBConnect from './databaseConnect'
```

Comment:
This is not all new, just that I've moved the file out of "actions" (since it is no longer relevant to just actions).

But it's a lot cleaner now, and won't run automatically when imported since the schedulers are now in a class.